src/deploy.cmd.js

```diff
@@ -176,18 +176,24 @@ class DeployCommand extends AbstractCommand {
 
     const ticks = {};
     const tick = (message, name) => {
+      const shortname = name.replace(/\/package.json.*/, '').replace(/node_modules\//, '');
       bar.tick({
-        action: name ? `packaging ${name}` : '',
+        action: name ? `packaging ${shortname}` : '',
       });
       if (message) {
-        this.log.log({
+        this.log.maybe({
           progress: true,
           level: 'info',
           message,
         });
       }
     };
 
+    if (this._dryRun) {
```

Comment:
no. this is important for testing (also manually). unless we separate `packaging` step from the `deploy` we need this.